atlp-rwanda/cabal-phantom-backend
# #usCaESGS Fixing setup multiple Language

### **What about this PR?**
This PR is about to fix setup multiple language and adding test coverage

### **Description of Task to be completed**
- Changing app language based on language used by computer
-  Add test coverage badge
- Move babel packages to dev dependencies
- Catch all unspecified routes

### **How should be manually tested?**
- Clone the repo
- Checkout the branch `fix-setup-multiple-language-usCaESGS`
- Install all packages
- About test with browser, you may change the browser's language and reload the app to see  the changes. (Optional)
- Test by using url: http://localhost:5000/?lang=rw
- To test unspecified route, use unspecified route. Example: http://localhost:5000/api/users

### **Relevant pivots tracker**
[#usCaESGS](https://trello.com/c/usCaESGS/20-support-multiple-languages)
